src/glass/src/app/core/top-bar/top-bar.component.ts

```diff
@@ -11,11 +13,25 @@ export class TopBarComponent implements OnInit {
   @Input('navigationSidenav')
   navigationSidenav!: MatSidenav;
 
-  constructor() {}
+  languages = {
+    /* eslint-disable @typescript-eslint/naming-convention */
+    en_GB: 'English',
```

Comment:
Oh... do we want to use `en_GB` not `en_US` instead?

Reply:
Why should we use US english, the brits have invented it :-)

Reply:
Because... well, at least I'm more used to it. But I'm fine with `en_GB` as well :wink: